frontend/src/component/playground/Playground/PlaygroundForm/PlaygroundConnectionFieldset/PlaygroundConnectionFieldset.tsx

```diff
@@ -68,6 +86,10 @@ export const PlaygroundConnectionFieldset: VFC<
 
     const { projects: availableProjects } = useProjects();
 
+    const isChangeRequestPlaygroundEnabled = useUiFlag(
```

Comment:
I'd use changeRequestPlaygroundEnabled since most of our flags drop the is... prefix in the UI code